feature-libs/product/configurators/rulebased/styles/_configurator-issues-notification.scss

```diff
@@ -1,42 +1,55 @@
 %cx-configurator-issues-notification {
-  .cx-error-container {
+  display: none;
```

Comment:
This file also somewhere under  /feature-libs/product/configurators/common/styles